appinfo/app.php

```diff
@@ -28,7 +28,7 @@
 		'order' => 3,
 		'href' => $g->linkToRoute('spreed.page.index'),
 		'icon' => $g->imagePath('spreed', 'app.svg'),
-		'name' => $l->t('Calls'),
+		'name' => $l->t('Video calls'),
```

Comment:
This name will get way too long when translated. Let’s use just »Calls« here please.